### PR TITLE
[4.0] Remove module description popover

### PR DESCRIPTION
**Pull request by brianteeman**
When you go to select a NEW module you have a list of all the modules with a very short 90 character truncated description.

If you hover over that description you get a popover with a short 200 character truncated description. This can result in a popover that is identical to the text it is supposed to be describing

This popover is not accessible - I doubt even most sighted users will know that it even exists as there is no visual indicator that it exists either.

The original reason for the 90 character truncation was back in the day that this was a modal and there was limited space. That is no longer true  but we do need to have some form of truncation as some 3pd write essays for the description ;) so I chose the 200 character limit

This PR removes the popover and changes the displayed description to the 200 character version

I am including the screenshots as I doubt most people even knew the popover existed
### Before
image

### After
image

### CODE REVIEW

File: administrator/components/com_modules/tmpl/select/default.php [hunk 1]
```diff
@@ -25,17 +24,15 @@
 	<?php $link       = 'index.php?option=com_modules&task=module.add&eid=' . $item->extension_id; ?>
 	<?php $name       = $this->escape($item->name); ?>
 	<?php $desc       = HTMLHelper::_('string.truncate', $this->escape(strip_tags($item->desc)), 200); ?>
-	<?php $short_desc = HTMLHelper::_('string.truncate', $this->escape(strip_tags($item->desc)), 90); ?>
```
Comment (Quy) [inline administrator/components/com_modules/tmpl/select/default.php:28]: Remove spaces before `=`.

**Comment by Quy** [comment]
I know this has nothing to do with this PR. Please confirm that the link should be on the right side.

@infograf768 

rtl

**Comment by brianteeman** [comment]
Strange. It was on the right in my test setting the XML value to right

**Comment by brianteeman** [comment]
I confirm your results with the farsi language
BUT
If you set 	`	<rtl>1</rtl>`
in the english language it works correctly

I am assuming there is an issue with this line of code 	`<?php if ($document->direction != 'rtl') : ?>
`
but that is unrelated to this pr

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8fc18492e5a4d94d6125f0059ea437dbb64e0a6c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24634">issues.joomla.org/tracker/joomla-cms/24634</a>.</sub>

**Comment by Quy** [comment]
Removing the `if` statement and using this code only, then it works in both cases.
```
	<li class="list-group-item">
		<a href="<?php echo Route::_($link); ?>" class="mr-2"><strong><?php echo $name; ?></strong></a>
		<small><?php echo $desc; ?></small>
	</li>
```

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 8fc18492e5a4d94d6125f0059ea437dbb64e0a6c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24634">issues.joomla.org/tracker/joomla-cms/24634</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by wilsonge** [comment]
So is there any way now of seeing the untruncated description in this view?

**Comment by brianteeman** [comment]
Same as before. In the module itself.

In the list view before we showed 90 characters and 200 on hover
Now we show 200.

**Comment by wilsonge** [comment]
Ahh sorry I missed we were only showing 200 in the popover

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
No problem. I was surprised at the limit in the popover too.

**Comment by infograf768** [comment]
@Quy 
Yes, the link should be on the right side.
We have the same error in 3.9.x
=> No use in this case to have a specific order for rtl.

```
diff --git a/administrator/components/com_modules/tmpl/select/default.php b/administrator/components/com_modules/tmpl/select/default.php
index 5157566..86a26df 100644
--- a/administrator/components/com_modules/tmpl/select/default.php
+++ b/administrator/components/com_modules/tmpl/select/default.php
@@ -25,16 +25,8 @@
 	<?php $name = $this->escape($item->name); ?>
 	<?php $desc = HTMLHelper::_('string.truncate', $this->escape(strip_tags($item->desc)), 200); ?>
-
-	<?php if ($document->direction != 'rtl') : ?>
 	<li class="list-group-item">
 		<a href="<?php echo Route::_($link); ?>" class="mr-2"><strong><?php echo $name; ?></strong></a>
 		<small><?php echo $desc; ?></small>
 	</li>
-	<?php else : ?>
-	<li class="list-group-item">
-		<small><?php echo $desc; ?></small>
-		<a href="<?php echo Route::_($link); ?>" class="mr-2"><strong><?php echo $name; ?></strong></a>
-	</li>
-	<?php endif; ?>
 <?php endforeach; ?>
 </ul>
```

**Comment by infograf768** [comment]
Code was OK in 2012. ;)

**Pull request merged**